torch/distributed/fsdp/_runtime_utils.py

```diff
@@ -1001,6 +1050,9 @@ def _post_backward_final_callback(
     root_state = state
 
     if root_state._sync_gradients:
+        # TODO (rohan-varma): this also waits for the overlapped optimizer step to finish
```

Comment:
Should we track somewhere the follow-ups (e.g. an issue)?

Reply:
@awgu tracking these here: https://github.com/pytorch/pytorch/issues/104479